ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1036,6 +1036,25 @@ where is_in_physics_bubble(vessel)
       plugin_state = "managing physics bubble";
     }
     UnityEngine.GUILayout.TextArea(text : "Plugin is " + plugin_state);
+    if (FlightGlobals.ActiveVessel != null) {
```

Comment:
Maybe a comment saying that this if statement should be removed once we properly handle collisions.